strawberry/schema/name_converter.py

```diff
@@ -53,7 +53,7 @@ def from_argument(self, argument: StrawberryArgument) -> str:
         return self.get_graphql_name(argument)
 
     def from_object(self, object_type: TypeDefinition) -> str:
-        if object_type.concrete_of:
+        if object_type.concrete_of and not object_type.is_generic_specialized:
```

Comment:
could we do this check inside `from_generic`?

Reply:
Sure. Just noticed that this change was not required. Removing it